expertise/create_dataset.py

```diff
@@ -46,22 +46,32 @@ def get_profile_ids(client, group_ids):
     :param group_ids: List of group ids
     :type group_ids: list[str]
 
-    :return: List of tilde ids
+    :return: List of tuples containing (tilde_id, email)
     :rtype: list
     """
     tilde_members = []
+    members = []
     for group_id in group_ids:
         group = openreview_client.get_group(group_id)
+
         profile_members = [member for member in group.members if '~' in member]
+        profile_search_results = openreview_client.search_profiles(emails=None, ids=profile_members, term=None)
+        if profile_search_results and type(profile_search_results) == dict:
```

Comment:
If you search by profile ids then the results would be a list instead of a dict. Is this being taken into account here @carlosmondra ?

Reply:
Woooo! Thank you very much for spotting this HUGE mistake. I didn't know that search_profiles returned different data types depending on the parameter. I have fixed that and removed some extra unnecessary checks.

Reply:
All the more reason to add tests. I think I'll need to work on those